### PR TITLE
107906740 Shop Owners Edit Products 

### PR DESCRIPTION
**Pull request by temi-adediran**
#### What does this PR do?

Enables shop owners to edit products.
#### Description of Task to be completed?

Shop owners should be able to edit products.
#### What are the relevant pivotal tracker stories?
#107906740

### CODE REVIEW

File: app/controllers/sessions_controller.rb [hunk 1]
```diff
@@ -2,54 +2,84 @@ class SessionsController < ApplicationController
...
   end
 
   private
 
-  def process_form_login
-    user = RegularUser.find_by_email(params[:session][:email])
-    return if user_is_nil?(user)
-    if user.authenticate(params[:session][:password]) && user.active
-      login_successful user
+  def logout_user(_user)
```
Comment (betkom) [inline app/controllers/sessions_controller.rb:35]: @andela-tfowotade why the `_` here?

File: app/services/message_service.rb [hunk 2]
```diff
@@ -0,0 +1,67 @@
+class MessageService
```
Comment (betkom) [inline app/services/message_service.rb:1]: @andela-tfowotade Why is this class added again in this PR.  You should always try to update your branch if a change is already on another branch. Please update your PR and remove all codes repeated in the PRs I've merged

File: app/controllers/sessions_controller.rb (hunk 1 above)
Comment (temi-adediran) [inline app/controllers/sessions_controller.rb:35]: Robocop changed it from `user` to `_user` because it complained that it's not being used elsewhere.

**Comment by betkom** [comment]
@andela-tfowotade have you fully tested this? Will wait until you demo this tomorrow before merging

**Pull request merged**